Tests/LibGfx/TestImageWriter.cpp

```diff
@@ -21,21 +23,22 @@
 #include <LibGfx/ImageFormats/WebPWriter.h>
 #include <LibTest/TestCase.h>
 
-static ErrorOr<NonnullRefPtr<Gfx::Bitmap>> expect_single_frame(Gfx::ImageDecoderPlugin& plugin_decoder)
+static ErrorOr<NonnullRefPtr<Gfx::Bitmap>> expect_single_frame(Gfx::ImageDecoderPlugin& plugin_decoder, bool is_gif)
 {
     EXPECT_EQ(plugin_decoder.frame_count(), 1u);
     EXPECT(!plugin_decoder.is_animated());
-    EXPECT(!plugin_decoder.loop_count());
+    EXPECT((is_gif && plugin_decoder.loop_count() == 1) || (!is_gif && !plugin_decoder.loop_count()));
```

Comment:
It looks wrong to me that the gif writer has different behavior here. loop_count has the same semantics for gif, apng, and animated webp: 0 means "loops indefinitely", other values mean "runs n times". Why does the gif writer write 1 here?

Reply:
The block is optional, and I don't encode it, what you are seeing is the default of the decoder which are arguably weird.

Reply:
Oh I see. Maybe the gif decoder test just shouldn't use expect_single_frame() then?